Makefile

```diff
@@ -92,6 +98,15 @@ L3_SRCDIR   := $(SRCDIR)
 L3_INCDIR   := $(INCDIR)
 TESTS       := test-use-cases
 
+# Symbols needed to integrate with LineOfCode (LOC) package
+LOCPACKAGE      := LineOfCode/loc
+LOCGENPY        := $(LOCPACKAGE)/gen_loc_files.py
+
+# Names of files generated by LOC Python generator
+LOC_FILENAMES   := loc_filenames.c
+LOC_HEADER      := loc.h
+LOC_TOKENS      := loc_tokens.h
```

Comment:
This chunk of definitions is where the LOC-package and its artifacts makes its appearance in L3-universe!!